server/src/models/Task.ts

```diff
@@ -0,0 +1,13 @@
+class Task {
+  private timeStart: number;
+
+  constructor(public task: string) {
+    this.task = task;
+    this.timeStart = new Date().getTime();
+  }
+
+  getTimeStamp() {
+    return new Date().getTime() - this.timeStart;
+  }
```

Comment:
To raczej nie potrzebne, nie bedziemy raczej odpytywali serwera zeby nam timer na froncie zauktualizowal (serwer by bardzo ucierpial jakby co kazda milisekunde mial wysylac info do frontu ze czas mija) :D